Description: Debian package specific changes
Author: Tzafrir Cohen <tzafrir@debian.org>
Description: Pathes that match Debian packaging

Forwarded: not-needed
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2013-09-25

--- plinth-0.0.0~git.20130925.orig/share/init.d/plinth
+++ plinth-0.0.0~git.20130925/share/init.d/plinth
@@ -11,11 +11,11 @@
 ### END INIT INFO
 
 # This file is /etc/init.d/plinth
-DAEMON=/usr/local/bin/plinth.py
+DAEMON=/usr/bin/plinth
 PID_FILE=/var/run/plinth.pid
 
-PLINTH_USER=www-data
-PLINTH_GROUP=www-data
+PLINTH_USER=plinth
+PLINTH_GROUP=plinth
 
 test -x $DAEMON || exit 0
 
